### PR TITLE
[BugFix]: get_check_constraints breaks and does not generate valid SQL

### PR DESCRIPTION
**Pull request by fabianmenges**
This fixes both issues.

### CODE REVIEW

File: sqla_vertica_python/vertica_python.py [hunk 1]
```diff
@@ -234,19 +234,19 @@ def get_check_constraints(self, connection, table_name, schema=None, **kw):
...
-                    v_catalog.tables i]
+                    v_catalog.tables i
                 WHERE
                     i.table_name='{table_name}'
-                AND
-                    cons.constraint_type = 'c'
                 {schema_clause}
             )
         """.format(table_name=table_name, schema_clause=(
-            "" if schema is None else "AND i.table_schema ='{schema}'".format(schema)))
```
Comment (fabianmenges) [inline sqla_vertica_python/vertica_python.py:249]: This python breaks with ``` in get_check_constraints
    "" if schema is None else "AND i.table_schema ='{schema}'".format(schema)))
KeyError: 'schema'```

File: sqla_vertica_python/vertica_python.py [hunk 2]
```diff
@@ -234,19 +234,19 @@ def get_check_constraints(self, connection, table_name, schema=None, **kw):
         FROM
             v_catalog.table_constraints cons
         WHERE
+            cons.constraint_type = 'c'
+          AND
             cons.table_id = (
                 SELECT
                     i.table_id
                 FROM
-                    v_catalog.tables i]
```
Comment (fabianmenges) [inline sqla_vertica_python/vertica_python.py:241]: Bad syntax

File: sqla_vertica_python/vertica_python.py [hunk 3]
```diff
@@ -234,19 +234,19 @@ def get_check_constraints(self, connection, table_name, schema=None, **kw):
...
+                    v_catalog.tables i
                 WHERE
                     i.table_name='{table_name}'
-                AND
-                    cons.constraint_type = 'c'
```
Comment (fabianmenges) [inline sqla_vertica_python/vertica_python.py:245]: Wrong where clause for this filter

**Comment by lemeryfertitta** [comment]
Looks good to me. Thanks.

**Pull request merged**